lib/datadog/tracing/contrib/extensions.rb

```diff
@@ -108,6 +108,46 @@ def configure(&block)
           module Settings
             InvalidIntegrationError = Class.new(StandardError)
 
+            def self.included(base)
+              base.class_eval do
+                settings :contrib do
+                  # Schema version for span attributes that enables various features. Only returns 'v0' for now
+                  #
+                  # @default `v0`
+                  # @return [String]
+                  option :span_attribute_schema do |o|
+                    o.default 'v0'
+                  end
```

Comment:
We only use this value on `spec/support/network_helpers.rb`. Should we remove the possibility of someone setting a value different `v0` altogether and leave the `v0` value hardcoded  on `spec/support/network_helpers.rb`? Mainly to avoid users using a setting that has no functionality yet. And avoid the possibility to have issues once we actually introduce different version 😄

Reply:
Implemented!